.github/workflows/docs.yml

```diff
@@ -85,6 +85,8 @@ jobs:
                       -r requirements/requirements.test.integration.txt \
                       -r requirements/requirements.http.txt \
                       -r requirements/requirements.cli.txt
+
+          pip install -e .
```

Comment:
Is this needed/desired?

Reply:
yes because otherwise the typer utils cant introspect the module to generate the inference cli docs